src/appMain/main.cc

```diff
@@ -125,6 +125,12 @@ int32_t main(int32_t argc, char** argv) {
     profile::Profile::instance()->config_file_name("smartDeviceLink.ini");
   }
 
+  // Unsibscribe once for all threads
+  if ( utils::UnsibscribeFromTermination() ) {
```

Comment:
@anosach-luxoft , because this function return result system API function and return 0 - in succces, and other positive numbers in other cases.